angular-workspace/projects/ni/nimble-angular/src/thirdparty/directives/checkbox_value_accessor.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * [Nimble]
+ * Copied from https://github.com/angular/angular/blob/035aee01089b9f9d4b5b6af66a74002e07723fba/packages/forms/src/directives/checkbox_value_accessor.ts
```

Comment:
Can you use the url that has the tag this was pulled from? Makes it a lot easier to see which version of Angular we last copied changes from.